packages/outputarea/src/widget.ts

```diff
@@ -128,6 +132,28 @@ export class OutputArea extends Widget {
    */
   readonly rendermime: IRenderMimeRegistry;
 
+  /**
+   * The hidden output models.
+   */
+  private trimmedOutputModels: IOutputModel[];
+
+  /*
+   * The maximum outputs to show in the trimmed
+   * output area.
+   */
+  private maxNumberOutputs: number;
+
+  /*
+   * The maximum outputs to show in the trimmed
+   * output head and tail areas.
+   */
+  private headTailNumberOutputs: number = -1;
+
+  /*
+   * The index for the end of the head in case of trim mode.
+   */
+  private headEndIndex: number = -1;
```

Comment:
Does the `-1` has any specific significance? In which case maybe add it in the docstring?

Reply:
-1 has a usage in previous iteration, but does not have anymore. It is remove. Thx for taking care @goanpeca  👍